java/java.lsp.server/vscode/package.json

```diff
@@ -407,6 +414,39 @@
 			{
 				"command": "nbls:Edit:org.openide.actions.DeleteAction",
 				"title": "Delete"
+			},
+			{
+				"command": "workbench.action.debug.run",
+				"title": "Run Without Debugging",
+				"icon": "$(run)"
+			},
```

Comment:
Do we want all these new commands to be visible in the Command Palette? If yes, consider adding the `category` attribute, if no, create corresponding items (containing "when": "false") under `menus/commandPalette`.

Reply:
Actually I think that actions like java.project.debug could be in the command palette ... they are able to work with editor URI to derive the target project, so I'll add categories to them. Not sure if I should use `Java`, or rather invent a `Project` ...

Reply:
Categories assigned in c3c3eb783c